virtcontainers/fc.go

```diff
@@ -537,6 +525,29 @@ func (fc *firecracker) createJailedDrive(name string) (string, error) {
 	return r, nil
 }
 
+// when running with jailer, firecracker binary will firstly be copied into fc.fcJailerRoot and
+// then being executed there. Therefore we need to ensure fc.JailerRoot has exec permissions.
+func (fc *firecracker) fcRemountWithExec(src string) error {
+	if err := fc.bindMount(context.Background(), src, src, false); err != nil {
```

Comment:
`fc.bindMount` just copies `bindMount` mount.go. Any reason not to use the common one (and remove the fc one)? Also it would be nice to put `fcRemountWithExec()` in mount.go as well.

Reply:
nvm, I see there is a minor difference -- fc bind mount might be a slave mount and wants to receive propagations. It makes sense for jailer to conveniently modify the jail house though I don't know if it is used that way.

Reply:
yep, you're right~~~ It should be a slave mount. Later, we will bind mount other resources(`vmlinux`, `rootfs`, etc) under jailerRoot and they should be propagated.